coretex/entities/sample/network_sample.py

```diff
@@ -137,6 +165,9 @@ def download(self, ignoreCache: bool = False) -> bool:
                 sampleHardLinkPath.unlink()
                 os.link(self.zipPath, sampleHardLinkPath)
 
+        if self.zipPath.exists():
```

Comment:
This check is not needed, let's remove it and call `os.utime` as a part of the previous `if` clause which checks if response was successful or not.